I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 8, in script call
    call ch1_main from _call_ch1_main
  File "game/script-ch1.rpy", line 217, in script
    fb "what!,hvordan skulle jeg svare rigtigt på det,det jo bare en random person som du kender!"
Exception: Sayer 'fb' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 8, in script call
    call ch1_main from _call_ch1_main
  File "game/script-ch1.rpy", line 217, in script
    fb "what!,hvordan skulle jeg svare rigtigt på det,det jo bare en random person som du kender!"
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 2579, in execute
    Say.execute(self)
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 581, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 470, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'fb' is not defined.

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.4.24120703
A Week Of School 2 1.0.0
Sun Dec 15 17:21:52 2024
